.circleci/config.yml

```diff
@@ -163,6 +182,35 @@ jobs:
       - store_artifacts:
           path: ~/project/android/app/build/outputs
 
+  release-android:
+    working_directory: ~/project/android
+    docker:
+      - image: circleci/android:api-27-node8-alpha
+    environment:
+      TERM: dumb
+      LC_ALL: en_US.UTF-8
+      LANG: en_US.UTF-8
+      APK_PATH: "./app/build/outputs/apk/production/release/app-production-release.apk"
+    steps:
+      - checkout:
+          path: ~/project
+      - attach_workspace:
+          at: ~/project
+      - <<: *android-release-secrets
+      - <<: *fabric-android-secrets
+      - <<: *restore-gradle-cache
+      - <<: *restore-gems-cache
+      - <<: *ruby-dependencies
+      - <<: *android-dependencies
+      - <<: *save-gradle-cache
+      - <<: *save-gems-cache
+      - run:
+            command: |
+                bundle exec fastlane release \
```

Comment:
Copy of beta-android job. change is apk path and call release lane which has different arguments